browser/components/CodeEditor.js

```diff
@@ -947,9 +948,11 @@ export default class CodeEditor extends React.Component {
     }
 
     const pastedTxt = clipboard.readText()
-
+    console.log(pastedTxt);
```

Comment:
I think you forgot to remove this line.

Reply:
Thank you for your review. I've just fixed it.